docs/source/getting-started/content/proc_starting-the-virtual-machine.adoc

```diff
@@ -37,5 +37,6 @@ $ {bin} start --vm-driver virtualbox --bundle _path_to_system_bundle_
 
 [NOTE]
 ====
-The cluster takes a minimum of four minutes to start the necessary containers and operators before serving a request.
+* The cluster takes a minimum of four minutes to start the necessary containers and operators before serving a request.
+* If errors are encountered on the start, check the Troubleshooting section for potential solutions. 
```

Comment:
Could you link (`xref`) the troubleshooting section here so that the reader doesn't have to go looking for it? :)

Reply:
I've added a link to the troubleshooting section